dhcpv4/dhcpv4.go

```diff
@@ -371,9 +371,32 @@ func (d *DHCPv4) SetUnicast() {
 // GetOption will attempt to get all options that match a DHCPv4 option
 // from its OptionCode.  If the option was not found it will return an
 // empty list.
+//
+// According to RFC 3396, options that are specified more than once are
+// concatenated, and hence this should always just return one option.
 func (d *DHCPv4) GetOption(code OptionCode) []Option {
+	return d.Options.GetOption(code)
+}
+
+// GetOneOption will attempt to get an  option that match a Option code.
+// If there are multiple options with the same OptionCode it will only return
+// the first one found.  If no matching option is found nil will be returned.
+func (d *DHCPv4) GetOneOption(code OptionCode) Option {
+	return d.Options.GetOneOption(code)
+}
+
+// Options is a collection of options.
+type Options []Option
+
+// GetOption will attempt to get all options that match a DHCPv4 option
+// from its OptionCode.  If the option was not found it will return an
+// empty list.
+//
+// According to RFC 3396, options that are specified more than once are
+// concatenated, and hence this should always just return one option.
+func (o Options) GetOption(code OptionCode) []Option {
```

Comment:
I'm sure I'll get confused by this `Options` vs `[]Option` in the future, but this is OK for me